charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -41,13 +46,25 @@ spec:
       resources:
         requests:
           storage: {{ .Values.cassandra.cassandraLibDirVolume.size | default "5Gi" }}
+    {{- if and (eq .Values.medusa.storage "local") ( .Values.medusa.podStorage ) }}
+    additionalVolumes:
+      - mountPath: /mnt/backups
+        name: medusa-backups
+        pvcSpec:
+          storageClassName: {{ .Values.medusa.podStorage.storageClass }}
+          accessModes:
+            - ReadWriteOnce
```

Comment:
Local backups would presumably be used with some sort of shared storage. Shouldn't we also support `ReadWriteMany`?

Reply:
Added support for modifying access modes.